src/test/scala/cromwell/SimpleWorkflowActorSpec.scala

```diff
@@ -30,7 +28,7 @@ class SimpleWorkflowActorSpec extends CromwellTestkitSpec("SimpleWorkflowActorSp
   val TestExecutionTimeout = 5.seconds.dilated
 
   "A WorkflowActor" should {
-    "start, run, succeed and die" in {
+    "start, run, succeed and die" ignore {
```

Comment:
The deadlocks may be because we're ~~abusing~~ using the synchronous parts of the Akka TestKit, instead of just using the regular actor references:

> Note
> It is highly recommended to stick to traditional behavioural testing (using messaging to ask the Actor to reply with the state you want to run assertions against), instead of using TestActorRef whenever possible.
>
> Due to the synchronous nature of TestActorRef it will not work with some support traits that Akka provides as they require asynchronous behaviours to function properly. Examples of traits that do not mix well with test actor refs are PersistentActor and AtLeastOnceDelivery provided by Akka Persistence.

-- http://doc.akka.io/docs/akka/2.3.12/scala/testing.html#Synchronous_Unit_Testing_with_TestActorRef

We generally don't inspect the internals of any of our actors, so they don't need to be a blocking, synchronous, `TestActorRef` / `TestFSMRef`. Switching the actor back to an asynchronous `ActorRef` seems to unblock the tests on my machine:

```scala
private def buildWorkflowFSMRef(sampleWdl: SampleWdl, rawInputsOverride: String): ActorRef = {
  val workflowSources = WorkflowSourceFiles(sampleWdl.wdlSource(), rawInputsOverride, "{}")
  val descriptor = WorkflowDescriptor(WorkflowId(UUID.randomUUID()), workflowSources)
  system.actorOf(WorkflowActor.props(descriptor, new LocalBackend))
}
```